### PR TITLE
Feature/new landing page

### PR DESCRIPTION
**Pull request by HenrikBischkopf**

### CODE REVIEW

File: pages/watering.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import WateringSchedule from "@/components/WateringSchedule";
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import BackArrow from "@/components/MyPlant/BackArrow";
+
+function useDayCount(startDay) {
+  const [day, setDay] = useState(startDay);
+  function handleNextDay() {
+    setDay(day + 1);
+  }
```
Comment (JessicaLoers) [inline pages/watering.js:10]: Consider to use a functional update for the state to ensure the latest state value is always used -  safety first 👮

```suggestion
function handleNextDay() {
    setDay((currentDay) => currentDay + 1); 
      }
```

File: pages/watering.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  const daysOfWeek = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
```
Comment (JessicaLoers) [inline pages/watering.js:29]: Consider to move this outside your component. Although the performance gain might be negligible in this specific case, moving static data outside of a component prevents it from being recreated on every render.

**Review by JessicaLoers — CHANGES_REQUESTED** [review]
Please pay attention to the use of the react fragments.

File: components/Tile.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  }
+`;
+
+const StyledIcon = styled(Image)`
+  margin-top: 10px;
+`;
+
+const StyledText = styled.p`
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
```
[the remark this answers is not in the document]
Comment (Fabi911) [reply components/Tile.js:40]: that's not final.

File: components/FertilizingSchedule.js [hunk 4]
```diff
@@ -9,14 +10,10 @@ export default function FertilizingSchedule({ plantsToFertilize }) {
...
-            {/* if there are more than two strings, they are separated by a comma */}
-            {plant.fertiliser_season.map((season, index, id) => (
-              <FertilizeSpan key={id}>
-                {season}
-                {index !== plant.fertiliser_season.length - 1 ? ", " : ""}
-              </FertilizeSpan>
-            ))}
+            Fertiliser season:{" "}
+            <FertilizeSpan>
+              {(fertilizedSeasons = plant.fertiliser_season.join(", "))}
```
Comment (JessicaLoers) [inline components/FertilizingSchedule.js:15]: nice 👍

File: pages/404.js [hunk 5]
```diff
@@ -1,3 +1,3 @@
 export default function errorpage() {
-  return <h1>Shit happens</h1>;
+  return <h1>Shit happens 💩</h1>;
```
Comment (JessicaLoers) [inline pages/404.js:2]: Super important change.

**Review by JessicaLoers — APPROVED** [review]
**Well done ✨**

**Pull request merged**